worlds/civ_6/Rules.py

```diff
@@ -0,0 +1,57 @@
+from typing import TYPE_CHECKING, List, Dict
+from BaseClasses import CollectionState
+from .Items import get_item_by_civ_name
+from .Data import get_boosts_data
+from .Enum import CivVICheckType
+from .ProgressiveDistricts import convert_items_to_have_progression
+
+from worlds.generic.Rules import forbid_item, set_rule
+
+
+if TYPE_CHECKING:
+    from . import CivVIWorld
+
+
+def create_boost_rules(world: 'CivVIWorld'):
+    boost_data_list = get_boosts_data()
+    boost_locations = [location for location in world.location_table.values() if location.location_type == CivVICheckType.BOOST]
+    for location in boost_locations:
+        boost_data = next((boost for boost in boost_data_list if boost.Type == location.name), None)
+        world_location = world.get_location(location.name)
+        forbid_item(world_location, "Progressive Era", world.player)
+        if not boost_data or boost_data.PrereqRequiredCount == 0:
+            continue
+
+        set_rule(world_location, lambda state, prereqs=boost_data.Prereq, required_count=boost_data.PrereqRequiredCount: has_required_items(state, prereqs, required_count, world))
```

Comment:
```suggestion
        if boost_data and boost_data.PrereqRequiredCount > 0:
            set_rule(world_location, lambda state, prereqs=boost_data.Prereq, required_count=boost_data.PrereqRequiredCount: has_required_items(state, prereqs, required_count, world))
```
I don't think this is a necessary change but idk I like it

Reply:
Yeah this makes more sense, updating 👍